portfolio/src/main/webapp/comments.js

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+window.onload = () => { 
+  fetch('/data').then(response => response.json()).then((comments) => {
+    console.log(comments);
+    const comments_list = document.getElementById('comments-container');
+    comments_list.innerHTML = '';
+    for(i=0; i!=comments.length; ++i){
```

Comment:
`i < comments.length` is more conventional when iterating through a list.

Alternatively, you can use for-each

```javascript
comments.forEach((comment) => {
    comments_list.appendChild(createCommentElement(comment.email,comment.comment));
})
```